test/e2e/autoscale_test.go

```diff
@@ -249,7 +227,8 @@ func TestAutoscaleUpDownUp(t *testing.T) {
 		clients.KubeClient,
```

Comment:
The log message about "Manually setting ScaleToZeroThreshold" a few lines above this is no longer relevant.

Reply:
Done.